server/controllers/expression-sheet-parser.js

```diff
@@ -3,28 +3,46 @@
 
 const EXPRESSION_SHEET_SUFFIXES = ["_expression", "_optimized_expression", "_sigmas"];
 
+var numbersToLetters = {0:"A", 1:"B", 2:"C", 3:"D", 4:"E", 5:"F", 6:"G", 7:"H", 8: "I", 9:"J", 10:"K", 11:"L",
+    12:"M", 13:"N", 14:"O", 15:"P", 16:"Q", 17:"R", 18:"S", 19:"T", 20:"U", 21:"V", 22:"W", 23:"X", 24:"Y",
+    25:"Z", 26:"AA", 27:"AB", 28:"AC", 29:"AD", 30:"AE", 31:"AF", 32:"AG", 33:"AH", 34:"AI", 35:"AJ", 36:"AK",
+    37:"AL", 38:"AM", 39:"AN", 40:"AO", 41:"AP", 42:"AQ", 43:"AR", 44:"AS", 45:"AT", 46:"AU", 47:"AV", 48:"AW",
+    49:"AX", 51:"AY", 52:"AZ", 53:"BA", 54:"BB", 55:"BC", 56:"BD", 57:"BE", 58:"BF", 59:"BG", 60:"BH", 61:"BI",
+    62:"BJ", 63:"BK", 64:"BL", 65:"BM", 66:"BN", 67:"BO", 68:"BP", 69:"BQ", 70:"BR", 71:"BS", 72:"BT", 73:"BU",
+    74:"BV", 75:"BW", 76:"BX"};
+
+
 const errorsList = {
-    idLabelError: function () {
+    idLabelError: function (sheetName) {
         return {
             errorCode: "MISLABELED_ID_CELL",
-            possibleCause: "The top left cell of the expression sheet is mislabeled.",
+            possibleCause: "The top left cell of the " + sheetName + " sheet is mislabeled.",
```

Comment:
Not a huge deal, but when you can, look up JavaScript template strings (the ones that are delimited by backticks **``**)—these can be very useful in building strings like these in a slightly more compact way.